cloudify-web-monitoring-plugin/plugin.yaml

```diff
@@ -0,0 +1,69 @@
+plugins:
```

Comment:
Let's put all plugin files in a subfolder named "plugin".

Reply:
done